docs/deployments/clerk-environment-variables.mdx

```diff
@@ -8,13 +8,17 @@ You can use environment variables to configure how your Clerk app behaves, such
 
 This page is a reference for all available Clerk environment variables.
 
+## Compatibility
+
+In the frontend, Clerk's environment variables work for most popular meta-frameworks, such as Next.js or Remix. If you're building a pure React app, you should use the props on the components you're using.
+
+For example, to force users to redirect to a specific URL after signing in, you would use the `signInForceRedirectUrl` prop on [`<ClerkProvider>`](/docs/components/clerk-provider) rather than the `CLERK_SIGN_IN_FORCE_REDIRECT_URL` environment variable.
```

Comment:
```suggestion
In the frontend, Clerk's environment variables work for most popular meta-frameworks, such as Next.js or Remix. 

If you're building a pure React app, you should use the props on the components you're using. For example, to force users to redirect to a specific URL after signing in, you would use the `signInForceRedirectUrl` prop on [`<ClerkProvider>`](/docs/components/clerk-provider) rather than the `CLERK_SIGN_IN_FORCE_REDIRECT_URL` environment variable.
```